include/zephyr/bluetooth/gap.h

```diff
@@ -38,18 +38,26 @@ extern "C" {
 #define BT_DATA_TX_POWER                0x0a /* Tx Power */
 #define BT_DATA_SM_TK_VALUE             0x10 /* Security Manager TK Value */
 #define BT_DATA_SM_OOB_FLAGS            0x11 /* Security Manager OOB Flags */
+#define BT_DATA_PERIPHERAL_INT_RANGE    0x12 /* Peripheral Connection Interval Range */
```

Comment:
This is technically called the Slace Connection Interval Range, but updated it with the new proper naming scheme, and will file an erratum.

Reply:
https://bluetooth.atlassian.net/browse/ES-22293

Reply:
Nice, thanks for catching this.